shared/chat/conversation/messages/message-popup/exploding/container.js

```diff
@@ -41,6 +43,11 @@ const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch, ownProps: OwnProps) => ({
+  _onCopy: (message: Types.Message) => {
```

Comment:
you have `message` available in `ownProps.message`, so this function doesn't need to take a param